sql/main_summary_v4.sql

```diff
@@ -0,0 +1,290 @@
+
+--
+-- Query generated by: templates/unnested_list_and_map_view.sql.py main_summary_v4 main_summary --view-dataset=telemetry_raw
```

Comment:
I think we have dataset names switched here. Should be:

    templates/unnested_list_and_map_view.sql.py main_summary_v4 main_summary --view-dataset=telemetry --table-dataset=telemetry_raw

Also, we may want to have separate flags for table project and view project, so that we can create views in derived-datasets that reference tables that have been moved to telemetry_raw in shared-prod. Or maybe we handle that manually? I don't have a strong opinion about what exactly gets codified in the repo for the transition period, which is hopefully pretty short.

Reply:
okay, updated options to be...better? please review